types/resolve/index.d.ts

```diff
@@ -44,6 +45,14 @@ type readFileCallback = (err: Error | null, file?: Buffer) => void;
  */
 type realpathCallback = (err: Error | null, resolved?: string) => void;
 
+/**
+ * Callback invoked when reading and parsing a package.json file
+ *
+ * @param error
+ * @param resolved Absolute path to the resolved file
+ */
+type readPackageCallback = (err: Error | null, package?: unknown) => void;
```

Comment:
can do something like `Record<string, unknown>` instead of `unknown` I guess. not sure if it provides any value or not

Reply:
"an object of unknown" is way more valuable than "unknown", that's for sure :-)